src/annotator/integrations/test/html-metadata-test.js

```diff
@@ -405,5 +405,20 @@ describe('HTMLMetadata', () => {
 
       assert.equal(doc.uri(), canonicalLink.href);
     });
+
+
+    it('should log an error if URI is not decodable', () => {
+      // '%%CUST_ID%%' is an invalid escape sequence, so it will throw a URIError when decoded
+      const badURI = 'https://example.com/?foo=%%CUST_ID%%' 
+      sinon.stub(testDocument, '_getDocumentHref').returns(badURI); 
```

Comment:
I missed this on my previous review, but stubbing/mocking methods from the subject under test should be avoided, because then you are not testing the actual implementation, but the mock.